dashboard/src/actions/charts.test.tsx

```diff
@@ -76,6 +76,66 @@ describe("fetchCharts", () => {
   });
 });
 
+describe("fetchChartCategories", () => {
+  it("fetches chart categories", async () => {
+    response = [{ id: "foo" }];
+    const expectedActions = [
+      { type: getType(actions.charts.requestChartsCategories) },
+      { type: getType(actions.charts.receiveChartCategories), payload: response },
+    ];
+    await store.dispatch(actions.charts.fetchChartCategories(cluster, namespace));
+    expect(store.getActions()).toEqual(expectedActions);
+    expect(axiosGetMock.mock.calls[0][0]).toBe(
+      `api/assetsvc/v1/clusters/${cluster}/namespaces/${namespace}/charts/categories`,
+    );
+  });
+
+  it("fetches chart categories by repos", async () => {
+    response = [{ id: "foo" }];
+    const expectedActions = [
+      { type: getType(actions.charts.requestChartsCategories) },
+      { type: getType(actions.charts.receiveChartCategories), payload: response },
+    ];
+    await store.dispatch(actions.charts.fetchChartCategories(cluster, namespace, "foo,bar"));
+    expect(store.getActions()).toEqual(expectedActions);
+    expect(axiosGetMock.mock.calls[0][0]).toBe(
+      `api/assetsvc/v1/clusters/${cluster}/namespaces/${namespace}/charts/categories?repos=foo,bar`,
+    );
+  });
```

Comment:
since you are not finally using the `repo` as an argument, you can remove this related code